config.json

```diff
@@ -1850,26 +1850,32 @@
   ],
   "key_features": [
     {
+      "icon": "fast",
       "title": "Performance",
       "content": "Rust is blazingly fast and memory-efficient; it has no runtime or garbage collector."
     },
     {
+      "icon": "stable",
       "title": "Reliability",
       "content": "Rust’s rich type system and ownership model guarantee memory-safety and thread-safety."
     },
     {
+      "icon": "productive",
```

Comment:
![`productive`](https://raw.githubusercontent.com/exercism/website-icons/main/key-features/productive.svg) `productive`

You may prefer e.g.:

![`evolving`](https://raw.githubusercontent.com/exercism/website-icons/main/key-features/evolving.svg) `evolving`